crates/codegen/parser/runtime/src/support/parser_result.rs

```diff
@@ -93,7 +93,7 @@ impl Match {
     pub fn is_full_recursive(&self) -> bool {
         self.nodes
             .iter()
-            .flat_map(|node| cst::Node::create_cursor(node, Default::default()))
+            .flat_map(|node| cst::Node::cursor_with_offset(node, Default::default()))
```

Comment:
nit: suggest replacing the couple of remaining `Default::default()` with the newly added `::ZERO`.

Reply:
Ah, missed those. Thanks!